components/Downloads/ChangelogModal/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { ArrowUpRightIcon, XMarkIcon } from '@heroicons/react/24/outline';
+import * as Dialog from '@radix-ui/react-dialog';
+import { useTranslations } from 'next-intl';
+import type { FC, PropsWithChildren, ReactNode, ComponentProps } from 'react';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+import Link from '@/components/Link';
+
+import styles from './index.module.css';
+
+type ChangelogModalProps = {
+  heading: string;
+  subheading: string;
+  avatars: ComponentProps<typeof AvatarGroup>['avatars'];
+  trigger: ReactNode;
+  children: ReactNode;
+};
+
+const ChangelogModal: FC<PropsWithChildren<ChangelogModalProps>> = ({
+  heading,
+  subheading,
+  avatars,
+  trigger,
+  children,
+}) => {
+  const t = useTranslations();
+
+  return (
+    <Dialog.Root>
+      <Dialog.Trigger asChild>{trigger}</Dialog.Trigger>
+      <Dialog.Portal>
```

Comment:
Fonts look different because they are opened with the Portal in a place that is not covered by the font information in Storybook

Reply:
Wdym? Is it not using open sans then? If not then we need to manually set font-open-sans on the wrapper or content class?

Reply:
Yes, Open Sans is not currently used. Even if we specify `font-open-sans` in the wrapper, we cannot style it because the token we use for the font is not in an element that contains this element

But we can try something like that;

```css
  [--font-open-sans:"Open_Sans"]
  font-open-sans
```

Alternatively, we can add it as a token to the `previewBody` located in `.storybook/main.ts`

The current HTML structure is as follows;

<img width="757" alt="Screenshot 2023-11-21 at 19 32 35" src="https://github.com/nodejs/nodejs.org/assets/14062599/fa9db1d2-6c41-42e5-b59f-3d57934e184e">

Reply:
> Alternatively, we can add it as a token to the previewBody located in .storybook/main.ts

Let me see if I can make a quick fix from my side.